server/services/WeekPhsaseService/weekPhaseInterval.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+const { WeekPhaseDate } = require("./weekPhaseDate");
+
+class WeekPhaseInterval {
+  constructor(id, startTime, endTime, handler) {
+    this.id = id;
+    this.startTime = startTime;
+    this.endTime = endTime;
+    this.handler = handler;
+  }
+
+  //METHODS
+  //Returns true if the parameter is between the start and end time of this phase
+  isDateWithinInterval(date){
+    let dateString = date.toString();
```

Comment:
Make this method accept as input a dayjs date. Copy the exact behavior of the comment I wrote a week ago.